[bambulab] out of range exception

I have successfully created a bambulab printer thing. 
After creating an item for the channel **Print Stage**, I observe the following exception in the log files

```
com.fasterxml.jackson.databind.JsonMappingException: Numeric value (2521540800) out of range of int (-2147483648 - 2147483647)
```

## Expected Behavior

The item with the link to the channel **Print Stage** should be updated.

## Current Behavior

The value of the item stays at `NULL`


## Context

It seems the exception comes from the `ip`:

```
        "net": {
            "conf": 16,
            "info": [
                {
                    "ip": 2521540800,
                    "mask": 16777215
                },
                {
                    "ip": 0,
                    "mask": 0
                }
            ]
        },
```
Full exception log:
```
com.fasterxml.jackson.databind.JsonMappingException: Numeric value (2521540800) out of range of int (-2147483648 - 2147483647)
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 334, column: 37] (through reference chain: pl.grzeslowski.jbambuapi.mqtt.Report["print"]->pl.grzeslowski.jbambuapi.mqtt.Report$Print["net"]->pl.grzeslowski.jbambuapi.mqtt.Report$Print$Net["info"]->java.util.ArrayList[0])
	at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:401) ~[?:?]
	at com.fasterxml.jackson.databind.JsonMappingException.wrapWithPath(JsonMappingException.java:372) ~[?:?]
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer._deserializeFromArray(CollectionDeserializer.java:381) ~[?:?]
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:246) ~[?:?]
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:30) ~[?:?]
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:543) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeWithErrorWrapping(BeanDeserializer.java:585) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:447) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1497) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:348) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185) ~[?:?]
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:543) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeWithErrorWrapping(BeanDeserializer.java:585) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:447) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1497) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:348) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185) ~[?:?]
	at com.fasterxml.jackson.databind.deser.SettableBeanProperty.deserialize(SettableBeanProperty.java:543) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeWithErrorWrapping(BeanDeserializer.java:585) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:447) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1497) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:348) ~[?:?]
	at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:185) ~[?:?]
	at com.fasterxml.jackson.databind.deser.DefaultDeserializationContext.readRootValue(DefaultDeserializationContext.java:342) ~[?:?]
	at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:4917) ~[?:?]
	at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:3921) ~[?:?]
	at pl.grzeslowski.jbambuapi.mqtt.PrinterWatcher.consume(PrinterWatcher.java:35) ~[?:?]
	at pl.grzeslowski.jbambuapi.mqtt.PrinterClient.lambda$connect$0(PrinterClient.java:105) ~[?:?]
	at java.util.ArrayList.forEach(Unknown Source) ~[?:?]
	at java.util.Collections$SynchronizedCollection.forEach(Unknown Source) ~[?:?]
	at pl.grzeslowski.jbambuapi.mqtt.PrinterClient.lambda$connect$1(PrinterClient.java:103) ~[?:?]
	at org.eclipse.paho.client.mqttv3.internal.CommsCallback.deliverMessage(CommsCallback.java:511) ~[?:?]
	at org.eclipse.paho.client.mqttv3.internal.CommsCallback.handleMessage(CommsCallback.java:417) ~[?:?]
	at org.eclipse.paho.client.mqttv3.internal.CommsCallback.run(CommsCallback.java:214) ~[?:?]
	at java.lang.Thread.run(Unknown Source) [?:?]
Caused by: com.fasterxml.jackson.core.exc.InputCoercionException: Numeric value (2521540800) out of range of int (-2147483648 - 2147483647)
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 334, column: 37]
	at com.fasterxml.jackson.core.base.ParserMinimalBase._reportInputCoercion(ParserMinimalBase.java:618) ~[?:?]
	at com.fasterxml.jackson.core.base.ParserMinimalBase.reportOverflowInt(ParserMinimalBase.java:680) ~[?:?]
	at com.fasterxml.jackson.core.base.ParserBase.convertNumberToInt(ParserBase.java:1052) ~[?:?]
	at com.fasterxml.jackson.core.base.ParserBase._parseIntValue(ParserBase.java:974) ~[?:?]
	at com.fasterxml.jackson.core.base.ParserBase.getIntValue(ParserBase.java:784) ~[?:?]
	at com.fasterxml.jackson.databind.deser.std.NumberDeserializers$IntegerDeserializer.deserialize(NumberDeserializers.java:526) ~[?:?]
	at com.fasterxml.jackson.databind.deser.std.NumberDeserializers$IntegerDeserializer.deserialize(NumberDeserializers.java:506) ~[?:?]
	at com.fasterxml.jackson.databind.deser.std.MapDeserializer._readAndBindStringKeyMap(MapDeserializer.java:623) ~[?:?]
	at com.fasterxml.jackson.databind.deser.std.MapDeserializer.deserialize(MapDeserializer.java:449) ~[?:?]
	at com.fasterxml.jackson.databind.deser.std.MapDeserializer.deserialize(MapDeserializer.java:32) ~[?:?]
	at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer._deserializeFromArray(CollectionDeserializer.java:361) ~[?:?]
	... 32 more

```

## Your Environment
* Version used: openHAB 5.0.0.M2

Thanks for the binding!

--- a/bundles/org.openhab.binding.bambulab/pom.xml
+++ b/bundles/org.openhab.binding.bambulab/pom.xml
@@ -18,7 +18,7 @@
     <dependency>
       <groupId>pl.grzeslowski</groupId>
       <artifactId>JBambuAPI</artifactId>
-      <version>1.5.0</version>
+      <version>1.6.0</version>
     </dependency>
     <dependency>
       <groupId>org.eclipse.paho</groupId>

